vulcan/lib/client/jsx/components/vulcan_nav.jsx

```diff
@@ -33,21 +32,18 @@ const Halo = ({radius}) => (
 );
 
 const Logo = connect(({exchanges}) => ({exchanges}))(({exchanges}) => (
-  <div
-    id='vulcan-logo'
-    className={Object.keys(exchanges).length > 0 ? 'throb' : null}
-  >
+  <div id='vulcan-logo' className='etna-vulcan-logo'>
     <div className='image' />
     <Halo radius={25} />
   </div>
 ));
 
 const getTabs = () => ({
-  workflows: Routes.workflow_path(CONFIG.project_name),
+  workflow: `/${CONFIG.project_name}/workflow`,
```

Comment:
I would much favor moving these to a constant (e.g. just a static version of the compiled routes file) rather than having them embedded all over in individual functions, which is pretty hard to track.

Reply:
Sounds good, I can change that. But right now I'm guessing there won't be many Vulcan routes, since I expect the UI to make calls directly to Archimedes...

Reply:
Will include this change in a subsequent PRs, so we can keep building.